deploy/helm/values.yaml

```diff
@@ -696,3 +696,7 @@ storageClass:
       fsType: xfs
     tke:
       volumeType: CLOUD_SSD
+
+
+rbac:
+  enabled: true
```

Comment:
default value shouldn't be enabled, or we need a complete user guides/documentation for RBAC ACL, that explain why we suggest this is enabled by default, and extra NOTES if it's disabled.